src/Settings.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Tribe\Extensions\Example;
+
+if ( ! class_exists( 'Settings' ) ) {
+	/**
+	 * Do the Settings.
+	 */
+	class Settings {
+
+		/**
+		 * The Settings Helper class.
+		 *
+		 * @var Settings_Helper
+		 */
+		protected $settings_helper = '';
+
+		/**
+		 * The prefix for our settings keys.
+		 *
+		 * @var string
+		 */
+		protected $opts_prefix = 'tribe_ext_example_';
+
+		public function __construct() {
+			$this->settings_helper = new Settings_Helper();
+
+			// Remove settings specific to Google Maps
+			add_action( 'admin_init', [ $this, 'remove_settings' ] );
+
+			// Add settings specific to OSM
+			add_action( 'admin_init', [ $this, 'add_settings' ] );
+		}
+
+		/**
+		 * Removes the settings from Events > Settings > General tab > "Map Settings" section
```

Comment:
Guessing this is here as an example, but can we make that clear by changing the comment to something like _"Here is an example of removing settings from..."_

Reply:
done